modules/todo.js

```diff
@@ -99,5 +100,17 @@ function isEarlierDate(todoDate, currentDate) {
 function getTodos() {
   return todos;
 }
+function saveTodosToLocalStorage() {
+  localStorage.setItem("todoArray", JSON.stringify(todos));
+}
+
+function loadTodosFromLocalStorage() {
+  todos.push(...JSON.parse(localStorage.getItem("todoArray")));
+
```

Comment:
Om man inte har någon todo i localstorage crashar programmet här. 

```suggestion
function loadTodosFromLocalStorage() {
  if (!localStorage.getItem("todoArray")) return;
  
  todos.push(...JSON.parse(localStorage.getItem("todoArray")));

```